components/service/glean/README.md

```diff
@@ -124,6 +127,33 @@ the following command can be used:
 
 `adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean.debug.GleanDebugActivity --ez logPings true`
 
+### Debug intent receiver
+
+Glean also contains an intent receiver that can be sent intents (messages) to
+perform certain actions in a running application.
+
+Users can trigger these intents using the following
+[`adb`](https://developer.android.com/studio/command-line/adb) command:
+
+`adb shell am broadcast -a {NAME_OF_INTENT} [extra keys]`
```

Comment:
Can we use this `broadcast` command for doing two actions at time? For example, sending pings and turning on logging on a single command? We will have an upcoming use-case for this, [see the upcoming proposal](https://docs.google.com/document/d/1w0aJK6AqqGAo_wr-iXMyBsmLYQTMKc5BXuSVkBS-MxU/edit?ts=5c6c335a#heading=h.h0ynf4clj2l3).

Reply:
We could put them as different arguments on the same intent, but I don't see a way to send multiple intents with the same broadcast command.

Reply:
This use case will be required for the debug view :( The other activity allows for that, since we use a single intent